src/movement/control_system.h

```diff
@@ -0,0 +1,156 @@
+#ifndef CONTROLSYSTEM_HPP
+#define CONTROLSYSTEM_HPP
+
+#include <cstdint>
+#include <eigen3/Eigen/Dense>
+#include <string>
+
+#include "geometry_msgs/Quaternion.h"
+#include "geometry_msgs/QuaternionStamped.h"
+#include "robosub/control.h"
+#include "robosub/control_status.h"
+#include "robosub/depth_stamped.h"
+#include "robosub/thruster.h"
+#include "ros/ros.h"
+#include "rotation_engine.hpp"
+#include "tf/transform_datatypes.h"
+
+using namespace Eigen;
+using std::string;
+
+namespace robosub
+{
+    class ControlSystem
+    {
+        /*
+         * Define custom types for using Eigen.
+         */
+        using Vector6d = Matrix<double,6,1>;
+        using Vector6 = Matrix<int,6,1> ;
+        using Vector12d = Matrix<double,12,1>;
+        using Matrix6d = Matrix<double,6,6>;
+
+        /*
+         * Defines conversion scale for converting between radians and degrees.
+         */
+        static constexpr double _PI_OVER_180 = (3.1415 / 180.0);
+        static constexpr double _180_OVER_PI = (180.0 / 3.14159);
```

Comment:
However, also note that these are class specific and those defined in rotation_engine are in the cpp, not the header. As of yet, they are all file-specific parameters.